dolphinscheduler-ui/src/js/conf/home/store/resource/actions.js

```diff
@@ -150,7 +150,7 @@ export default {
    */
   updateContent ({ state }, payload) {
     return new Promise((resolve, reject) => {
-      io.post(`resources/${payload.id}/update-content`, payload, res => {
+      io.post(`resources/${payload.id}`, payload, res => {
```

Comment:
It seems just need to change this method from post to put?

Reply:
It doesn't need to change the http method, but just change the url in the code(in dolphinscheduler-ui) to match the backend.

Reply:
@Liiyue Are you sure? it seems this bug is imported by 
- #6110
and you can check the payload, I am not sure.
https://github.com/apache/dolphinscheduler/blob/c1496d8a74ef69253d1873602a467ca61876bbfb/dolphinscheduler-ui/src/js/conf/home/pages/resource/pages/file/pages/edit/index.vue#L79-L86

BTW, we use PUT instead of POST, when we want to submit a update request

Reply:
You are right, the bug is caused by the confusion between PUT and POST. I have update the code for the bug and the UT in ResourcesControllerTest.

Reply:
Also,I have fix the mismatch of resource download url between frontend and backend.